lib/rspec/core/ordering.rb

```diff
@@ -58,6 +58,14 @@ def jenkins_hash_digest(string)
         MAX_32_BIT = 4_294_967_295
       end
 
+      # @private
+      # Orders items by modification time (most recent modified first).
+      class ModificationTime
+        def order(list)
+          list.sort_by { |item| -File.mtime(item.metadata[:absolute_file_path]).to_i }
```

Comment:
Do you think it makes any sense to optimize this? I guess ordering examples/groups inside the same spec file doesn't make much sense, and it would be possible to return just the `list` as the `Identity` (defined order) does.

Reply:
Does this still make sense?

Reply:
For sure we can leave this optimization for later if some project with 100k decides to use `recently-modified` and experiences some slow-downs.